src/view/MainFrame.cpp

```diff
@@ -204,9 +203,9 @@ void MainFrame::handleLBSel(int index) {
 			list_tabs.at(i).second->SetBackgroundColour(wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHT));
 			list_tabs.at(i).second->SetForegroundColour(wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHTTEXT));
 		} else {
-			list_tabs.at(i).first->SetBackgroundColour(*wxWHITE);
-			list_tabs.at(i).second->SetBackgroundColour(*wxWHITE);
-			list_tabs.at(i).second->SetForegroundColour(*wxBLACK);
+			list_tabs.at(i).first->SetBackgroundColour(wxNullColour);
+			list_tabs.at(i).second->SetBackgroundColour(wxNullColour);
+			list_tabs.at(i).second->SetForegroundColour(wxNullColour);
```

Comment:
is this still necessary then?

Reply:
This branch is necessary to remove the highlighting from the items again.

Thanks to your question I actually found that setting the background colour on `wxStaticText` does not seem to have any effect here. So I'll remove the second statement of each branch.

Reply:
:+1: